3rdPartyCypress/cypress/integration/worldServiceCoUkRedirect.js

```diff
@@ -0,0 +1,39 @@
+import services from '../../../src/app/lib/config/services';
+
+Object.keys(services).forEach(index => {
+  const service = index;
+
+  const notWSServices = ['news', 'cymrufyw', 'naidheachdan', 'default']; // Not WS
+
+  if (notWSServices.includes(service)) {
+    return;
+  }
+  describe('WS Redirects:', () => {
+    it(`all routes should redirect for ${service}`, () => {
+      const urlsTotest = [
+        `https://www.bbc.co.uk/${service}`,
+        `https://www.bbc.co.uk/${service}/articles/a00000000o`,
+        `http://www.bbc.co.uk/${service}/articles/a00000000o`,
+        `https://www.bbc.co.uk/${service}/articles/a00000000o.amp`,
+        `http://www.bbc.co.uk/${service}/articles/a00000000o.amp`,
```

Comment:
I don't think the `http` links are needed here, since this test is specifically focused on the `.co.uk`->`.com` redirect. 
```suggestion
```